### PR TITLE
Lk add starsolo barcode metrics

### PR DESCRIPTION
**Pull request by ekiernan**
### Description
We've had requests to include STARsolo per barcode metrics as an output to the Optimus workflow. This PR creates a new output (aligner_metrics) that includes the combined per barcode metrics across all shards to the Starsolo alignment task. 
----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [x] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: tasks/skylab/StarAlign.wdl [hunk 1]
```diff
@@ -389,6 +412,15 @@ task STARsoloFastq {
...
     File matrix_sn_rna = "matrix_sn_rna.mtx"
+    File cell_reads = "CellReads.stats"
+    File align_features = "Features.stats"
+    File summary = "Summary.csv"
+    File umipercell = "UMIperCellSorted.txt"
+    File cell_reads_sn_rna = "CellReads_sn_rna.stats"
+    File align_features_sn_rna = "Features_sn_rna.stats"
+    File summary_sn_rna = "Summary_sn_rna.csv"
+    File umipercell_sn_rna = "UMIperCellSorted_sn_rna.txt"
+    #File aligner_metrics = "~{output_bam_basename}.star_metrics.tar"
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply tasks/skylab/StarAlign.wdl:423]: Thanks for catching it. I'm going to delete it entirely.

**Review by nikellepetrillo — APPROVED** [review]

File: pipelines/skylab/optimus/Optimus.wdl [hunk 2]
```diff
@@ -245,6 +247,7 @@ workflow Optimus {
     File gene_metrics = GeneMetrics.gene_metrics
     File? cell_calls = RunEmptyDrops.empty_drops_result
     File? picard_metrics = DropseqMetrics.metric_output
+    File? aligner_metrics = MergeStarOutputs.cell_reads_out
```
Comment (kayleemathews) [inline pipelines/skylab/optimus/Optimus.wdl:250]: Hey Liz - would you mind adding the new output to the [Optimus Overview doc Outputs table](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Optimus_Pipeline/README.md#9-outputs)?

Comment (ekiernan) [reply pipelines/skylab/optimus/Optimus.wdl:250]: Thanks @kayleemathews ! Added it.

File: tasks/skylab/StarAlign.wdl [hunk 3]
```diff
@@ -337,31 +338,52 @@ task STARsoloFastq {
       --soloUMIdedup 1MM_Directional_UMItools \
       --outSAMtype BAM SortedByCoordinate \
       --outSAMattributes UB UR UY CR CB CY NH GX GN \
-      --soloBarcodeReadLength 0
```
Comment (jessicaway) [inline tasks/skylab/StarAlign.wdl:340]: Do we typically run star twice? Is it possible to run it just once?

**Review by jessicaway — APPROVED** [review]
Looks great! Thanks @ekiernan!

File: pipelines/skylab/optimus/Optimus.wdl (hunk 2 above)
Comment (kayleemathews) [reply pipelines/skylab/optimus/Optimus.wdl:250]: Thank you!

File: tasks/skylab/StarAlign.wdl (hunk 3 above)
Comment (ekiernan) [reply tasks/skylab/StarAlign.wdl:340]: We run it twice for the count_exons = true. With the latest version of STAR, we should be able to run both parameters in a single STAR run. We haven't yet played with that functionality.

**Comment by ekiernan** [comment]
retest this please

**Pull request merged**